src/main/java/org/gridsuite/merge/orchestrator/server/MergeOrchestratorService.java

```diff
@@ -144,28 +144,34 @@ public void consume(Message<String> message) {
 
                 mergeEventService.addMergeIgmEvent(processConfig.getProcess(), processConfig.getBusinessProcess(), dateTime, tso, IgmStatus.AVAILABLE, null, null, null, null, null);
             });
 
             if (!matchingProcessConfigList.isEmpty()) {
-                // import IGM into the network store
-                List<BoundaryInfos> lastBoundaries = cgmesBoundaryService.getLastBoundaries();
-                UUID networkUuid = caseFetcherService.importCase(caseUuid, lastBoundaries);
-                List<UUID> lastBoundariesUuid = toUuidBoundaries(lastBoundaries);
+                try {
+                    // import IGM into the network store
+                    List<BoundaryInfos> lastBoundaries = cgmesBoundaryService.getLastBoundaries();
+                    UUID networkUuid = caseFetcherService.importCase(caseUuid, lastBoundaries);
+                    List<UUID> lastBoundariesUuid = toUuidBoundaries(lastBoundaries);
 
-                LOGGER.info("Import case {} using last boundaries uuids {}", caseUuid, lastBoundariesUuid);
+                    LOGGER.info("Import case {} using last boundaries uuids {}", caseUuid, lastBoundariesUuid);
 
-                // check IGM quality
-                boolean valid = igmQualityCheckService.check(networkUuid);
+                    // check IGM quality
+                    boolean valid = igmQualityCheckService.check(networkUuid);
 
-                merge(matchingProcessConfigList.get(0), dateTime, date, tso, valid, networkUuid, caseUuid, null, null, lastBoundariesUuid);
+                    merge(matchingProcessConfigList.get(0), dateTime, date, tso, valid, networkUuid, caseUuid, null, null, lastBoundariesUuid);
 
-                for (ProcessConfig processConfig : matchingProcessConfigList.subList(1, matchingProcessConfigList.size())) {
-                    // import IGM into the network store
-                    UUID processConfigNetworkUuid = caseFetcherService.importCase(caseUuid, lastBoundaries);
-                    merge(processConfig, dateTime, date, tso, valid, processConfigNetworkUuid, caseUuid, null, null, lastBoundariesUuid);
+                    for (ProcessConfig processConfig : matchingProcessConfigList.subList(1, matchingProcessConfigList.size())) {
+                        // import IGM into the network store
+                        UUID processConfigNetworkUuid = caseFetcherService.importCase(caseUuid, lastBoundaries);
+                        merge(processConfig, dateTime, date, tso, valid, processConfigNetworkUuid, caseUuid, null, null, lastBoundariesUuid);
+                    }
+                } catch (Exception e) {
+                    ProcessConfig processConfig = matchingProcessConfigList.get(0);
+                    mergeEventService.addMergeIgmEvent(processConfig.getProcess(), processConfig.getBusinessProcess(), dateTime, tso, IgmStatus.VALIDATION_FAILED, null, null, null, null, null);
```

Comment:
Maybe we could distinguish between a failed import in the network store, and a validation that failed on the imported network in the network store, with two different igm status ?

Maybe in another US ....

Reply:
Yes we should do (IMPORT_FAILED)
An another US